HerPublicWebsite.BusinessLogic/Models/Questionnaire.cs

```diff
@@ -30,6 +30,29 @@ public record Questionnaire
 
     public ContactDetails ContactDetails { get; set; }
 
-    public bool EpcTooHigh => 
+    public bool FoundEpcBandIsTooHigh => 
         EpcDetails is { EpcRating: EpcRating.A or EpcRating.B or EpcRating.C };
+
+    public EpcRating EffectiveEpcBand
+    {
+        get
+        {
+            if (EpcDetails is null)
+            {
+                return EpcRating.Unknown;
+            }
+
+            if (EpcDetailsAreCorrect is not true)

```

Comment:
I've not seen this syntax before! Do we not prefer `if (!EpcDetailsAreCorrect)` for booleans? My `is` and `is not` preference is only for `null`-checks (admittedly I didn't think you could use it for anything else!)

Reply:
It's a nullable bool so it would have to be `EpcDetailsAreCorrect != true`. I don't have a strong preference on this, but after your earlier comments I think that this reads more nicely.